pandas/core/series.py

```diff
@@ -3887,7 +3887,12 @@ def f(x):
 
         with np.errstate(all="ignore"):
             if isinstance(f, np.ufunc):
-                return f(self)
+                try:
```

Comment:
pls do. it make changes here
core/apply.py is better

but to be honest i am not sure we need support this case as it’s invalid